tools/sggolangcilint/golangci.yml

```diff
@@ -1,7 +1,9 @@
 run:
   timeout: 10m
-  skip-dirs:
```

Comment:
Can this cause a problem for consumers? Should they be able to set this without having the entire config?

Reply:
Very good point! I'm updating so that the user can pass `...args string` to run and use flags to override specific stuff

Reply:
👍

Reply:
Updated!